### DISCUSSION TITLE
Unable to send address bytes...and other issues

### DISCUSSION THREAD

**Discussion by npaisnel** [HeyuX10Automation/heyu#64] [Q&A]
What could have suddenly have caused this.

Got hey set up on a RPi with HA-Bridge and a CM11

Last night going to nee .."Alexa, Every thing off"  worked perfectly ..if a little slow...

This morning ..no response..

If I SSH in and type command I get
 `pi@x10:/etc/heyu $ heyu on a1
Unable to send address bytes

pi@x10:/etc/heyu $ 
`

Other commands issued at SSH terminal 

`pi@x10:/etc/heyu $ heyu info
exread(): write failed
exread(): write failed
exread(): write failed
Invalid status response (was 0 bytes instead of 14)
HEYU: No response from the CM11A on /dev/ttyUSB0
Program exiting.

pi@x10:/etc/heyu $ heyu upload
Heyu directory /etc/heyu/ must be writable - quitting.
pi@x10:/etc/heyu $ 
`
I just checked permissions of etc/heyu

`drwxrwxrwx 3 root root    4096 Apr  8 07:58 heyu`

the etc/heyu directory is full of tmp files
named heyu.xxxx.tmp
zero bite serial numbered files
New one created each time a command issued

Any clues what this might be?

Been working fine for years..

**Comment by jkrzyszt** [comment]
That doesn't sound like an issue with the soruce code.

I know heyu-users yahoo group is missing and no other widely recognized service exists to submit your Heyu user support requests to, but please don't use github Issues for that.  If on github, please try uisng Discussions, maybe that service can take over the role of heyu-users group over time.

Thanks,
Janusz

**Comment by npaisnel** [comment]
I had not seen anything on here to indicate that this area was just for source code issues...The tab just says "issues" ..not "Source code issues" .  Sorry if I got it wrong.

It is titled "issues" and this is an issue I am having with it.
